resources/test/conftest.py

```diff
@@ -222,7 +222,7 @@ def _run_functional_test_variant(self, variant):
             # one
             for obj in [summarized, self.expected]:
                 obj["summarized_asserts"].sort(
-                    key=lambda x: (x["test"], x["status"], x["assert_name"], tuple(x["args"])))
+                    key=lambda x: (x["test"] or "None", x["status"], x["assert_name"], tuple(x["args"])))
```

Comment:
Oh that's an error now. I wonder if it would be better to use the empty string rather than the literal "`None`"

Reply:
Sgtm, done.